.github/workflows/release.yml

```diff
@@ -0,0 +1,72 @@
+name: relase-workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+        npmTag:
+          description: 'NPM tag to release'
+          required: true
+          default: 'next'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+    env:
+      CI: true
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup node
+        uses: actions/setup-node@v2-beta
+        with:
+          always-auth: true
+          node-version: 12
+          scope: '@lingui'
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install dependencies if needed
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile --pure-lockfile
+
+      - name: Unit Testing
+        run: yarn test
+
+      - name: Linting & Types
+        run: yarn lint:all
+
+      - name: Build packages
+        run: yarn release:build
+
+      - name: Integration Testing
+        run: yarn test:integration
+
+      - name: Creates local .npmrc
+        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+
+      - name: Prepare Lingui-Bot git account
+        uses: oleksiyrudenko/gha-git-credentials@v1
```

Comment:
What does this action do? Can't we use a `GH_TOKEN` for authentication?

Reply:
This is basically doing

git config --add user.email=linguibot@gmail.com
git config --add user.pwd=github password account

We can't use lingui bot gh token, because will override the internal github token of the ci.

Reply:
By default, if we use the github token, the commit will be like this:


INPUT_EMAIL=${INPUT_EMAIL:-'github-action@users.noreply.github.com'}
INPUT_NAME=${INPUT_NAME:-'GitHub Action'}

Reply:
Alright, I've just added `GH_BOT_PWD` to secrets. I just feel very unsecure by adding this password anywhere :/

Reply:
If you want we can do it manually, doing the config.set without any external action. It's pretty simple

Reply:
That's fine. Let's ship it :) Can't wait to see it in action! 👍